recipes/itak/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "2.0.1" %}
+{% set name = "itak" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lab.moilab.net/download/iTAK-{{ version }}.tar.gz
+  sha256: 24179ec86b6bde2e7ab99ef94c9eeefd18ed1b73d753482f9ff30f2080d88283
+
+build:
+  number: 0
+  noarch: python  #conflict with skip: true  # [not linux or not x86_64]
+  run_exports:
+    weak:
+      - {{ pin_subpackage('itak', max_pin="2.9.0") }}
```

Comment:
Is there a reason setting the max pin to 2.9.0?

Reply:
I changed it to 2.0.1 and also updated the meta.yaml with the other two changes you mentioned. I am not sure which one caused the error. anyway, the current meta file passed the check. Thanks a lot.